spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -773,6 +774,16 @@ def do_nothing():
             triggered=self.show_preferences_dialog,
             register_action=False
         )
+        self.close_action = self.create_action(
+            name=ArrayEditorActions.Close,
+            icon=self.create_icon('close_pane'),
+            text=_('Close'),
+            triggered=self.reject,
+            shortcut=self.get_shortcut(ArrayEditorActions.Close),
```

Comment:
```suggestion
            shortcut=self.get_shortcut(ArrayEditorActions.Close),
            register_action=False,
```

This is necessary to prevent a warning to be shown in the Internal console.